src/olympia/lib/settings_base.py

```diff
@@ -1050,6 +1050,8 @@ def get_db_config(environ_var, atomic_requests=True):
 VALIDATION_FAQ_URL = ('https://wiki.mozilla.org/Add-ons/Reviewers/Guide/'
                       'AddonReviews#Step_2:_Automatic_validation')
 
+SHIELD_STUDIES_SUPPORT_URL = 'https://support.mozilla.org/kb/shield'
```

Comment:
cc @mythmon - that's the page I'll redirect users to when hitting shield-study 2-16, the pioneer page is being kept intact.